scripts/gen-nncp.sh

```diff
@@ -331,7 +331,7 @@ EOF_CAT
         cat >> ${DEPLOY_DIR}/${WORKER}_nncp.yaml <<EOF_CAT
       ipv4:
```

Comment:
We should add IPv6 here, if not I am worried this will break the IPv6 test environment.

Reply:
it has an IPv6 section, just like the other networks, it's on  line 345, it's just hidden by github UI because the line did not change. Now, I'm not sure if the subnet should have a particular value, but it's there

Reply:
sorry, I missed that. 
Is changing the IPv6 prefix for `octavia` ok?
Prior to this change it was `fd00:eeee::` it is `fd00:eded::` after this change.

The `fd00:eeee::` is also in several other files - so it is going to break octavia?
```
scripts/gen-nncp.sh:        - ip: fd00:eeee::${IPV6_ADDRESS_SUFFIX}
scripts/gen-service-kustomize.sh:          - end: fd00:eeee::250
scripts/gen-service-kustomize.sh:            start: fd00:eeee::100
scripts/gen-service-kustomize.sh:          cidr: fd00:eeee::/64
scripts/gen-netatt.sh:        "range": "fd00:eeee::/64",
scripts/gen-netatt.sh:        "range_start": "fd00:eeee::30",
scripts/gen-netatt.sh:        "range_end": "fd00:eeee::70"
out/openstack/nncp/cr/api.sno.lab.example.com_nncp.yaml:        - ip: fd00:eeee::5
out/openstack/infra/cr/kustomization.yaml:          - end: fd00:eeee::250
out/openstack/infra/cr/kustomization.yaml:            start: fd00:eeee::100
out/openstack/infra/cr/kustomization.yaml:          cidr: fd00:eeee::/64
```

Reply:
I missed that, thanks, it's probably wrong, I'll swap the subnets for octavia and storage_mgmt